.github/workflows/release-automated.yml

```
@@ -1,79 +1,79 @@
-          labels: ${{ steps.meta.outputs.labels }}
+name: release-automated
+on:
+  push:
+    branches: [ release, alpha, beta, next-major ]
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    outputs:
+      current_tag: ${{ steps.tag.outputs.current_tag }}
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)